Author: Milan P. StaniÄ‡ <mps@arvanta.net>
Date:   Sat May  3 12:45:50 CEST 2025

Standard macro should be LLONG_MAX
thanks to NRK on #musl IRC on libera.chat

--- a/src/fops_misc.c	2025-05-02 08:45:02.000000000 +0000
+++ b/src/fops_misc.c	2025-05-03 09:22:01.929254612 +0000
@@ -935,7 +935,7 @@
 	{
 		char *t;
 		long long l;
-		if((l = strtoll(p + 1, &t, 10)) > 0 && t[1] == '\0' && l != LONG_LONG_MAX)
+		if((l = strtoll(p + 1, &t, 10)) > 0 && t[1] == '\0' && l != LLONG_MAX)
 		{
 			len = p - result;
 			i = l;
